.dockerignore

```diff
@@ -1,3 +1,11 @@
-node_modules
```

Comment:
I think `node_modules` should be ignored. I see there was a similar conversation on this comment https://github.com/poanetwork/token-bridge/pull/109#discussion_r227427251